packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1267,3 +1276,65 @@ export function restoreControlledState(
       return;
   }
 }
+
+export function listenToEventResponderEvents(
+  eventResponder: ReactEventResponder,
+  element: Element | Document,
+): void {
+  if (enableEventAPI) {
+    const {targetEventTypes} = eventResponder;
+    // Get the listening set for this element. We use this to track
```

Comment:
just a nit, but I think we should standardise on using capital S Set, else it's easy to get wrong (especially for noobs like me)
```suggestion
    // Get the listening Set for this element. We use this to track
```